addons/events/index.js

```diff
@@ -1,60 +1,8 @@
-const AddonsApi = require('modulor-storybook/addons');
+const AddonsApi = require('../../addons');
```

Comment:
please, no paths like this. it's unstable. that's why it was `'modulor-storybook/addons'`

Reply:
but then does it mean it is using itself as dependency?

Reply:
yes, why not?

Reply:
this way you can move this directory all around, no matter how many levels down

Reply:
makes sense. I'll update this and next place.